packages/sdk/src/react/hooks.ts

```
@@ -89,205 +95,178 @@ export function useEntityId(...keys: BigNumberish[]): BigNumberish {
+                return null;
+            }
 
-        const [initialData, subscription] = await sdk.subscribeEntityQuery({
-            query: fetchingRef.current!,
-            callback: ({ data, error }) => {
-                if (data) {
-                    const entity = data.pop();
-                    if (entity && entity.entityId !== "0x0") {
-                        state.updateEntity(entity);
+            const [initialData, subscription] = await config.subscribeMethod({
+                query: fetchingRef.current!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)